src/input_parsers/OnnxParser.cpp

```diff
@@ -353,15 +353,15 @@ int getRequiredIntAttribute( onnx::NodeProto &node, String name )
     return attr->i();
 }
 
-const onnx::TensorProto getTensorAttribute( onnx::NodeProto &node, String name )
+const onnx::AttributeProto *getTensorAttribute( onnx::NodeProto &node, String name )
```

Comment:
Turns out making copy of TensorProto can be inefficient and caused the CI to time out.